app/src/main/java/chat/rocket/android/renderer/MessageRenderer.java

```diff
@@ -35,7 +35,7 @@ public MessageRenderer avatarInto(ImageView imageView, String hostname) {
         imageView.setImageResource(R.drawable.ic_error_outline_black_24dp);
         break;
       default:
-        userRenderer.avatarInto(imageView, hostname);
+        userRenderer.avatarInto(imageView, hostname, object.getAvatar());
```

Comment:
I'll open (make public) the `getTextDrawable` from `Avatar` (it's the may reason to delegate any rendering to it).